cnxepub/scripts/collated_single_html/main.py

```diff
@@ -34,9 +34,12 @@ def main(argv=None):
     parser.add_argument('-d', '--dump-tree', action='store_true',
                         help='Print out parsed model tree.')
 
-    parser.add_argument('-o', '--output', type=argparse.FileType('w'),
+    parser.add_argument('-o', '--output', type=argparse.FileType('w+'),
```

Comment:
And that's a lie. Turns out that it didn't extend properly, foo2 has the copied resources but _not_ the original content.